gno.land/cmd/gnoland/secrets.go

```diff
@@ -72,3 +76,84 @@ func constructSecretsPath(nodeDir string) string {
 		config.DefaultSecretsDir,
 	)
 }
+
+// printKeyValue searches and prints the given key value in JSON
+func printKeyValue[T *secrets | *config.Config](
```

Comment:
Can this function go in `config.go`, like `getFieldAtPath?`

Reply:
Moved:

[b5be3c5](https://github.com/gnolang/gno/pull/2393/commits/b5be3c5d45b4e4efd0f0ada29164086f0b776441)